Dockerfile.rosbag

```diff
@@ -0,0 +1,48 @@
+FROM ros:galactic-ros1-bridge
+ENV DEBIAN_FRONTEND noninteractive
+
+# ROS
+ENV ROS1_DISTRO noetic
+ENV ROS2_DISTRO galactic
+
+# ROS2
+RUN set -x && \
+  sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list' && \
```

Comment:
This may be unnecessary.
See https://github.com/osrf/docker_images/blob/master/ros/galactic/ubuntu/focal/ros1-bridge/Dockerfile#L6.

Reply:
I did not delete the above line as it gave below build error.
**Build error:**
```
E: Unable to locate package ros-galactic--image-transport
E: Unable to locate package ros-galactic--cv-bridge
E: Unable to locate package ros-galactic--rosbag2-bag-v2-plugins
E: Unable to locate package ros-galactic--ros2bag
E: Unable to locate package ros-galactic--rosbag2-transport
E: Unable to locate package ros-galactic--rosbag2-storage-default-plugins
```
To solve above, i had to bring back below line:
`sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list' && \`

Now it is working.

Reply:
@ymd-stella 
Deleted above line as suggested and rechecked, build is through and i am able to use ros2 bag play info command.
Sorry for the confusion.

Uploaded final Dockerfile.rosbag for review.